### ISSUE TITLE
Do you have resnet50-wholebody weight?

### ISSUE THREAD

**Issue by twmht** [openpifpaf/openpifpaf#458]
Hi,

From the document there are only shufflenet family (-checkpoint=shufflenetv2k16-wholebody or --checkpoint=shufflenetv2k30-wholebody)

Do you have some other weight like resnet50?

**Comment by DuncanZauss** [comment]
Hi,
Sorry, currently we do not have a pretrained resnet 50 available. If you have a powerful GPU, you can train your own model with a ResNet50 backbone. This would be the command: `python3 -m openpifpaf.train --lr=0.0001 --momentum=0.95 --b-scale=3.0 --epochs=150 --lr-decay 130 140 --lr-decay-epochs=10 --batch-size=16 --weight-decay=1e-5 --dataset=wholebody --wholebody-upsample=2 --basenet=resnet50 --loader-workers=16 --wholebody-train-annotations=<PathToYourMSCOCO>/data-mscoco/annotations/person_keypoints_train2017_wholebody_pifpaf_style.json --wholebody-val-annotations=<PathToYourMSCOCO>/data-mscoco/annotations/person_keypoints_val2017_wholebody_pifpaf_style.json --wholebody-train-image-dir=<COCO_train_image_dir> --wholebody-val-image-dir=<COCO_val_image_dir>`

**Comment by twmht** [comment]
@DuncanZauss 

that's great.

thank you

**Comment by twmht** [comment]
@DuncanZauss 

For the limited gpu memory I change resnet50 to mobilenetv2, the loss is quite large (over 40000) at the beginning.

is is normal? In my opinion, providing the training log is good for other people to reproduce the result.  Do you have training log?
